swagger.yaml

```diff
@@ -0,0 +1,999 @@
+swagger: "2.0"
+info: 
+  title: Pick And Drop App
+  description: This application serves individuals and companies who are into logistic businesses, 
+    that is, the business of picking up an item and dropping it at a transport company Park for waybill 
+    or the Home of the receiver if within the locality. The application allows companies/individuals to 
+    create accounts and pick-up officers/offices. To create an order( request for pickup services), 
+    the sender will fill out their details, the receiver's details, and the order details. 
+    As the Order moves from one point to another, its status is updated.
+  version: 1.0.0
+  licence:
+    name: MIT
+    url: https://opensource.org/licenses/MIT
+  servers:
+    - url: https://pick-and-drop.onrender.com
+      description: Production Host
+    - url: http://localhost:5000
+      description: Local Host Server
+  contact:
+    name: Dozie Udeagha
+    email: dozie.udeagha@gmail.com
+
+schemes: [https, http]
+basePath: /api/v1
+paths:
+  /auths/register/company:
+    post:
+      summary: Create a company
```

Comment:
You can use tags to group related resources - [ref](https://swagger.io/docs/specification/grouping-operations-with-tags/)